apps/builder/app/builder/features/style-panel/style-source/style-source-input.tsx

```diff
@@ -64,18 +64,31 @@ export type ItemSelector = {
   state?: string;
 };
 
+const TextFieldContainer = styled("div", {
+  // Custom
+  display: "flex",
+  flexWrap: "wrap",
+  alignItems: "center",
+  backgroundColor: theme.colors.backgroundControls,
+  gap: theme.spacing[3],
+  p: theme.spacing[3],
+  borderRadius: theme.borderRadius[4],
+  minHeight: theme.spacing[12],
+  minWidth: 0,
+  border: `1px solid ${theme.colors.borderMain}`,
+  "&:focus-within": {
+    outline: `2px solid ${theme.colors.borderFocus}`,
```

Comment:
Need `outlineOffset: -1` to hide the grey the border:
 
<img width="140" alt="Screenshot 2023-05-06 at 12 10 21" src="https://user-images.githubusercontent.com/825702/236614950-eb0d1d4e-381c-45ec-9fc7-36996e373ba4.png">

Reply:
Nice!